var/spack/repos/builtin/packages/pfapack/package.py

```diff
@@ -0,0 +1,64 @@
+# Copyright 2013-2020 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack import *
+
+
+class Pfapack(MakefilePackage):
+    """A library for numerically computing the Pfaffian of
+       a real or complex skew-symmetric matrix. This is based on
+       computing the tridiagonal form of the matrix under
+       unitary congruence transformations."""
+
+    homepage = "https://michaelwimmer.org/downloads.html"
+    url      = "https://michaelwimmer.org/pfapack.tgz"
+
+    version('2014-09-17', sha256='b68fc35dda23ee24c358641b1a92ef701c4ffa0b3f0b0808b24e68afeb58ef07')
+
+    depends_on('lapack')
+    depends_on('blas')
+    depends_on('python')
+    depends_on('py-numpy')
+    depends_on('py-scipy')
```

Comment:
These probably need to be `type=('build', 'run')`

Reply:
Thank you for your checking.
I will add them and push the modified recipe later.